code/src/components/FirstQuestion.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+// This is the firstquestion thats comes up and asks for the usersname.
+
+const FirstQuestion = ({
+  nameInput,
+  onNameInputChange,
+  onStepChange,
+  onEnter,
+}) => {
+  return (
+    <form className="question-container">
```

Comment:
I got warning message in chrome development tool says "Form submission canceled because the form is not connected", maybe it is because there is no submit function specified in this line?